src/main/java/eu/deltasource/internship/abankingsystem/model/BankInstitution.java

```diff
@@ -0,0 +1,84 @@
+package eu.deltasource.internship.abankingsystem.model;
+
+import eu.deltasource.internship.abankingsystem.enums.ExchangeRate;
+import eu.deltasource.internship.abankingsystem.enums.Taxes;
+
+import java.util.*;
+
+public class BankInstitution {
+
+    private final String name;
+
+    private final String address;
+
+    private final ArrayList<Owner> customers = new ArrayList<>();
+
+    private final Map<Taxes, Double> priceList; // (that describes all taxes and fees and all currency exchange rates)
+
+    private final Map<ExchangeRate, Double> exchangeRates;
+
+    private final List<BankAccount> bankAccounts = new ArrayList<>();
+
+    private int dayCountTime = 1;
+
+    public BankInstitution(String name, String address, HashMap<Taxes, Double> priceList, Map<ExchangeRate, Double> exchangeRate) {
+        this.name = name;
+        this.address = address;
+        this.priceList = priceList;
+        this.exchangeRates = exchangeRate;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public ArrayList<Owner> getCustomers() {
+        return customers;
```

Comment:
return unmodifiable list and also the return type should be List not ArrayList

Reply:
Done.